docs/JMS-batchsink.md

```diff
@@ -0,0 +1,57 @@
+# JMS Streaming Source
+
+
+Description
+-----------
+Produces JMS messages of different types as Message, Text, Bytes and Map to a specified Queue or Topic.
+
+Use Case
+--------
+Use this JMS Sink plugin when you want to produce messages to a JMS Queue/Topic. 
+
+
+Properties
+----------
+**Connection Factory**: Name of the connection factory. If not specified *ConnectionFactory* if considered by default.
```

Comment:
`If not specified *ConnectionFactory* if considered by default.` - what does this mean?

Reply:
Fixed. The second *if* (typo)  makes this sound confusing. 

It means: If not specified, the value *ConnectionFactory* is considered by default.